crypto/keys/keybase.go

```diff
@@ -179,6 +180,12 @@ func (kb dbKeybase) List() ([]Info, error) {
 	iter := kb.db.Iterator(nil, nil)
 	defer iter.Close()
 	for ; iter.Valid(); iter.Next() {
+		key := string(iter.Key())
+
+		if strings.HasSuffix(key, ".address") {
```

Comment:
this needs a comment IMHO

Reply:
I'd say if one needs to write code like ^^^, then something is wrong. Just a earlier warning to core devs.

Reply:
++